homeassistant/components/light/isy994.py

```diff
@@ -69,6 +69,10 @@ def turn_on(self, brightness=100, **kwargs) -> None:
             _LOGGER.debug('Unable to turn on light.')
 
     @property
+    def state_attributes(self):
```

Comment:
You need to add the comment for this since its public.   Something like "Flag Supported Attributes" seems appropriate.